crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -1817,3 +1817,10 @@ pub fn squeeze_config(curr: &Node) -> Vec<i64> {
 
     axes
 }
+
+pub fn one_hot_config(curr: &Node) -> (usize, [f32; 2], i64) {
+    let depth = curr.inputs[1].value.clone().unwrap().into_i64();
```

Comment:
We should add a better error message if we cannot read the tensor values (i.e., not constant):
```suggestion
    let depth = curr.inputs[1]
        .value
        .clone()
        .expect("OneHot: Only constant depth is currently supported")
        .into_i64();

    let values = curr.inputs[2]
        .value
        .clone()
        .expect("OneHot: Only constant on/off values is currently supported")
        .into_f32s();
```